src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -138,7 +142,7 @@ export class StudentProfilePageComponent implements OnInit {
           }, (response: ErrorMessageOutput) => {
             this.statusMessageService.showErrorMessage(response.error.message);
           });
-    });
+    }).then(() => {});
```

Comment:
Nope. Promise does not work like that. You need to provide `onReject` handler as the second parameter of the first `.then`.